### ISSUE TITLE
sphinx-quickstart blank prompts in PowerShell

### ISSUE THREAD

**Issue by LemonPi** [sphinx-doc/sphinx#5294]
Subject: using sphinx-quickstart with PowerShell (v1.0) in Windows results in blank prompts

### Problem
![image](https://user-images.githubusercontent.com/5508542/44043664-68ad23ec-9ef1-11e8-9edc-b1977b21ba30.png)

#### Procedure to reproduce the problem
run `sphinx-quickstart`

#### Expected results
Works fine in git-bash
![image](https://user-images.githubusercontent.com/5508542/44043810-bfa1048e-9ef1-11e8-9311-6e2de76b8286.png)

### Environment info
- OS: Windows 10
- Python version: 3.6.5
- Sphinx version: 1.7.6

**Comment by LemonPi** [comment]
This seems to be an unfortunate selection of font colour for the default PowerShell colouring scheme since I can select and copy the "blank" prompts...

**Comment by tk0miya** [comment]
Thank you for reporting.
Could you try this script in PowerShell console?
```
python -c 'from sphinx.util.console import purple; print(purple("Hello Sphinx"))'
```

At present, Sphinx uses purple for prompt of quickstart. If it is conflicted with PowerShell, we have to change it...

**Comment by LemonPi** [comment]
@tk0miya Hey thanks for the quick response. `print` parses the colouring as text and we get garbage (in standard white colour): `[35mHello Sphinx[39;49;00m`

ANSI colour escape sequences don't seem to be working (they just print the escape sequence) - I've tried some other colours manually.

Using `colorama` works though

![image](https://user-images.githubusercontent.com/5508542/44044906-fc0c8094-9ef4-11e8-85dc-aa3d8238d444.png)

**Comment by LemonPi** [comment]
OK yeah the previous garbage printing was because `colorama` wasn't initialized. Initializing with `color_terminal()` we can see that purple needs to be changed.

![image](https://user-images.githubusercontent.com/5508542/44045153-ad9d3100-9ef5-11e8-8351-0d7abe8b5cdb.png)

**Comment by tk0miya** [comment]
Thank you for confirmation. Sorry, I'd perfectly forgotten about colorama.
Okay, let's change the color!

**Comment by LemonPi** [comment]
36 (cyan) seems to be a decent colour for PowerShell and cmd.
It's not easy to select a colour!

![image](https://user-images.githubusercontent.com/5508542/44045552-d575bfac-9ef6-11e8-8b78-f66d33aac9f1.png)

**Comment by tk0miya** [comment]
Fixed by #5312. It makes the questions bold font.
Thank you for reporting.
